### ISSUE TITLE
Bottomless CLI is not updated on Crates

### ISSUE THREAD

**Issue by avinassh** [tursodatabase/libsql#857]
This is some very old version of bottomless CLI - https://crates.io/crates/bottomless-cli

We should update our workflow to publish newer versions automatically

**Comment by psarna** [comment]
yeah, 100%, we just need to figure out the dependency mess we have at the moment -- because we need all our dependencies (including forks) to be published on crates as well
